packages/conf-openjdk/conf-openjdk.1/opam

```diff
@@ -13,12 +13,13 @@ depexts: [
   ["java-latest-openjdk-devel"] {os-distribution = "centos"}
   ["jdk-openjdk"] {os-distribution = "arch"}
   ["openjdk"] {os-distribution = "homebrew" & os = "macos"}
+  ["openjdk21@edgecommunity"] {os-distribution = "alpine"}
+  ["Microsoft.OpenJDK.21"] {os-distribution != "cygwin" & os-family = "windows"}
```

Comment:
We don't have ability to depend on winget packages as part of depext - this might be better instead as a Windows-specific `post-message` below, a la the FreeBSD message

```
"You may need to first install a version of OpenJDK, e.g. winget install Microsoft.OpenJDK.21" {os = "win32" & failure}
```